pygfx/materials/_points.py

```diff
@@ -238,5 +238,34 @@ class PointsGaussianBlobMaterial(PointsMaterial):
     """
 
 
+class PointsSpriteMaterial(PointsMaterial):
+    """A material to render points as sprite images.
+
+    Renders the provided texture at each point position. The images are square
+    and sized just like with a PointMaterial. The texture color is multiplierd
+    with the point's "normal" color (as calculated depending on ``color_mode``).
+
+    The sprite texture is provided via ``.map``.
+    """
+
+    def __init__(self, *, sprite=None, **kwargs):
+        super().__init__(**kwargs)
+        self.sprite = sprite
+
+    @property
+    def sprite(self):
+        """The texture map specifying the sprite image.
+
+        The dimensionality of the map must be 2D. If None, it just shows a
+        uniform color.
+        """
+        return self._sprite
```

Comment:
should this be stored on store?

Reply:
Good point! I copied from `PointsMaterial.map`, but that one was faulty too :)

Reply:
I think you missed `self._sprite` though?

Reply:
VSCode refused to save the file because of conflicts with the typo-change :P